lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1264,6 +1264,7 @@ LowererMD::LowerEntryInstr(IR::EntryInstr * entryInstr)
         {
             IR::Instr * instrAdd = IR::Instr::New(Js::OpCode::ADD, fpOpnd, spOpnd, IR::IntConstOpnd::New(fpOffset, TyMachReg, this->m_func), this->m_func);
             insertInstr->InsertBefore(instrAdd);
+            Legalize(instrAdd);
```

Comment:
Use Lowerer::InsertAdd instead which should take care of legalization.

Other places as well.

Reply:
Changed places to use InsertAdd/InsertLea as appropriate where possible.